zomes/chat/src/validation.rs

```diff
@@ -3,38 +3,74 @@ use crate::message::Message;
 /// This is the current structure of the payload the holo signs
 #[hdk_entry(id = "joining_code_payload")]
 #[derive(Clone)]
-struct JoiningCodePayload {
-    role: String,
-    record_locator: String
+pub(crate) struct JoiningCodePayload {
+    pub role: String,
+    pub record_locator: String
+}
+
+/// check to see if this is the valid read_only membrane proof
+pub(crate) fn is_read_only_proof(mem_proof: &MembraneProof) -> bool {
+    let b = mem_proof.bytes();
+    b.len() == 1 && b[0] == 0
```

Comment:
Is it possible to simply compare this to `[0]`?
```suggestion
    b == &[0]
```

Reply:
yep